### ISSUE TITLE
[bug] Comments sometimes won't load

### ISSUE THREAD

**Issue by micahlt** [Scratch-Client-4/itchy-ionic#55]
Context: https://github.com/Scratch-Client-4/itchy-ionic/pull/46#issuecomment-1053727474 - second paragraph

Comments will not load that are:

- Disabled
- Past the first "page" returned by the Scratch API

**Comment by micahlt** [comment]
#58 will resolve the second part of this issue (past the 1st page).

**Comment by micahlt** [comment]
Reports that this is still not fixed (#61).  I'll look into it - it's possible I just haven't pushed the latest changes to Google Play.

**Comment by locness3** [comment]
there's a google play version?

**Comment by micahlt** [comment]
> there's a google play version?

Yep: https://play.google.com/store/apps/details?id=org.scratchclient4.itchy

**Comment by locness3** [comment]
> it's possible I just haven't pushed the latest changes to Google Play.

Yeah, changes from #58 were not released at the time.

**Comment by locness3** [comment]
What's the status of this?

**Comment by micahlt** [comment]
Still haven't had time to reproduce, and probably won't for ~1-2 months.  I'm moving to college on August 17th and it's a whirlwind until then.
